utils.go

```diff
@@ -90,8 +90,15 @@ func NextPowerOfTwo(v uint64) uint {
 	return uint(v)
 }
 
-func GetSlotTimestamp(beaconState *deneb.BeaconState, blockHeader *phase0.BeaconBlockHeader) uint64 {
-	return beaconState.GenesisTime + uint64(blockHeader.Slot)*12
+func GetSlotTimestamp(beaconState *spec.VersionedBeaconState, blockHeader *phase0.BeaconBlockHeader) uint64 {
+	var genesisTime uint64
+	switch beaconState.Version {
+	case spec.DataVersionDeneb:
+		genesisTime = beaconState.Deneb.GenesisTime
```

Comment:
there's no GenesisTime()

also do u want to put this in versioned_beacon_state.go

Reply:
added to versioned_beacon_state.go